components/infobox/commons/infobox_campaign.lua

```diff
@@ -25,7 +25,7 @@ function Campaign:createInfobox()
 	local args = self.args
 
 	local widgets = {
-		Header{name = self.name, image = args.image},
+		Header{name = self.name, image = args.image, imageDark = args.imagedarkmode},
```

Comment:
I know that this has already become the sort of standard, but I would really prefer it we kept convention, and used

```suggestion
		Header{name = self.name, image = args.image, imageDark = args.imagedark or args.imagedarkmode},
```